swagger.yml

```diff
@@ -832,7 +832,52 @@ definitions:
               description: The order ID or an order's package's tracking number associated with the event
             eventDescription:
               type: string
-              description: The description of the event
+              description: |-
+                The description of the events.
+                
+                The `eventDescription` is a human-readable version of the `eventName` Below is a mapping of the event with its corresponding description(s).
```

Comment:
Per the [ticket](https://boltlogistics.atlassian.net/browse/COPE-786), we want to update the docs with all the events and corresponding descriptions. Not sure if there's a better way to display this, but just did it as a list 🤷 

<img width="716" alt="Screenshot 2024-07-10 at 5 14 50 PM" src="https://github.com/SecondCloset/parcel-api-docs/assets/29466297/7bdfc4fe-8b18-4b94-99d3-3fe5b871c9df">